File: C++/TablaDeMultiplicarN.cpp
#include <iostream>

using namespace std;

int main()
{
    int num;
    cin >> num;

    for (int i = 1; i < 11; i++)
    {
        cout << num << "X" << i << "=" << num * i << endl;
    }

    return 0;
}

// https://omegaup.com/arena/problem/Tabla-de-multiplicar-N/

File: Java/TablaDeMultiplicarN.java
import java.io.*;

public class TablaDeMultiplicarN {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

    public static void main(String[] args) throws Exception {

        int num = Integer.parseInt(br.readLine());

        for (int i = 0; i < 11; i++) {
            System.out.println(num + "x" + i + "=" + num * i);
        }
    }
}

// https://omegaup.com/arena/problem/Tabla-de-multiplicar-N/